appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/xml/PersistenceTagNames.java

```diff
@@ -40,5 +40,7 @@ public interface PersistenceTagNames extends TagNames {
     public static String TRANSACTION_TYPE = "transaction-type";
     public static String SHARED_CACHE_MODE = "shared-cache-mode";
     public static String VALIDATION_MODE = "validation-mode";
+    public static String SCOPE = "scope";
+    public static String QAULIFIER = "qualifier";
```

Comment:
:eyes: q**ua** or q**au**?

Reply:
My guess would be "qua" being the right one, I'll change it